src/routes/+page.svelte

```diff
@@ -155,4 +299,31 @@
 		position: absolute;
 		transition: transform 0.1s linear;
 	}
+	/* reducing the opacing when the images overlap with the section text
+	for better readability */
+	@media (max-width: 850px) {
+		img.visible {
+			opacity: 0.5;
+			transform: translateY(0);
+		}
+	}
+	@media (min-width: 851px) {
+		img.visible {
+			opacity: 1;
+			transform: translateY(0);
+		}
+	}
```

Comment:
nit: since you already have the `@media (max-width: 850px)` defined, you don't really need a separate media query for `@media (min-width: 851px)`, you can just include it directly and it will be applied for screen sizes greater than `850px` :)

So you can remove the media query wrapper and include just this
```
img.visible {
	opacity: 1;
	transform: translateY(0);
}
```

Reply:
this also means you can remove the `transform: translateY(0);` from the `max-width` media query, since it will already be applied

Reply:
ooh yeah I forgot to take into account ordering here! updating!